server/app.js

```diff
@@ -76,6 +74,8 @@ module.exports = function createApp({ signInService }) {
 
   // Routing
   app.use(authenticationMiddleware, csrfTokenMiddleware)
+  app.use(setUpMaintenance())
```

Comment:
the `authenticationMiddleware` sets up `employeeName` and `authUrl` so needs to happen after that